train_model.py

```diff
@@ -176,73 +122,73 @@ def train_model(
     print(f"Saving weight files in {weights_dir}")
 
     # Initialize tokenizers directory
-    tokenizers_dir = Path(data_dir) / "tokenizers"
-    tokenizers_dir.mkdir(parents=False, exist_ok=True)
-    print(f"Saving tokenizer files in {tokenizers_dir}")
+    # tokenizers_dir = Path(models_path) / "tokenizers"
+    # tokenizers_dir.mkdir(parents=False, exist_ok=True)
+    # print(f"Saving tokenizer files in {tokenizers_dir}")
 
     # Create SummaryWriter to record logs for TensorBoard
-    if tboard_dir is None:
-        tboard_log_dir = Path(data_dir) / "logs" / model_label
+    if config.tboard_path is None:
+        tboard_log_dir = Path(config.models_path) / "logs" / model_label
     else:
-        tboard_log_dir = f"{tboard_dir}/logs/{model_label}"
+        tboard_log_dir = f"{config.tboard_path}/logs/{model_label}"
     writer = SummaryWriter(log_dir=tboard_log_dir)
     print(f"Saving TensorBoard logs in {tboard_log_dir}")
 
     # Save all the variables in args as JSON inside folder
     json_string = json.dump(
-        obj=arg_dict,
+        obj=arg_table,
```

Comment:
Double check if arg_dict is used for anything else. If no, remove arg_dict (created at start of method.)

Reply:
It's used up above for something else.